<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1432</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1154</width>
    <height>495</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа 13</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QTabWidget" name="page1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>300</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="Slit_page">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <attribute name="title">
              <string>Дифракция на щели</string>
             </attribute>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QFrame" name="Slit">
                <property name="minimumSize">
                 <size>
                  <width>1080</width>
                  <height>421</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QFrame" name="frame_4">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>L =</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_2">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                     </widget>
                    </item>
                    <item alignment="Qt::AlignLeft">
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>м                                                b =</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                     </widget>
                    </item>
                    <item alignment="Qt::AlignLeft">
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>Нм</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton">
                      <property name="text">
                       <string>Показать</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_3">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QLabel" name="label_9">
                      <property name="text">
                       <string/>
                      </property>
                      <property name="pixmap">
                       <pixmap>img-qo4c_Q.png</pixmap>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QFrame" name="frame">
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_3">
                       <item>
                        <widget class="QChartView" name="graphicsView">
                         <property name="mouseTracking">
                          <bool>true</bool>
                         </property>
                         <property name="backgroundBrush">
                          <brush brushstyle="SolidPattern">
                           <color alpha="255">
                            <red>0</red>
                            <green>0</green>
                            <blue>0</blue>
                           </color>
                          </brush>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QFrame" name="frame_5">
                         <property name="frameShape">
                          <enum>QFrame::StyledPanel</enum>
                         </property>
                         <property name="frameShadow">
                          <enum>QFrame::Raised</enum>
                         </property>
                         <layout class="QHBoxLayout" name="horizontalLayout">
                          <item>
                           <widget class="QLabel" name="label_3">
                            <property name="text">
                             <string>TextLabel</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QLabel" name="label_4">
                            <property name="text">
                             <string>TextLabel</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tabWidgetPage2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <attribute name="title">
              <string>Дифракция на одномерной решетке</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QFrame" name="frame_6">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QFrame" name="frame_10">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QLabel" name="L">
                      <property name="text">
                       <string>L =</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_5"/>
                    </item>
                    <item>
                     <widget class="QLabel" name="b">
                      <property name="text">
                       <string>м                              b =</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_4"/>
                    </item>
                    <item>
                     <widget class="QLabel" name="N">
                      <property name="text">
                       <string>Нм                                 N =</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit_3"/>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_2">
                      <property name="text">
                       <string>Показать</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_8">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                     <widget class="QLabel" name="label_8">
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>99999</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="pixmap">
                       <pixmap>дифреш2.jpg</pixmap>
                      </property>
                      <property name="scaledContents">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QFrame" name="frame_9">
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_5">
                       <item>
                        <widget class="QChartView" name="graphicsView_2">
                         <property name="mouseTracking">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QFrame" name="frame_7">
                         <property name="frameShape">
                          <enum>QFrame::StyledPanel</enum>
                         </property>
                         <property name="frameShadow">
                          <enum>QFrame::Raised</enum>
                         </property>
                         <layout class="QHBoxLayout" name="horizontalLayout_5">
                          <item>
                           <widget class="QLabel" name="label_7">
                            <property name="text">
                             <string>TextLabel</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QLabel" name="label_6">
                            <property name="text">
                             <string>TextLabel</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action">
   <property name="text">
    <string>Дифракция на щели</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Дифракция на однородной решетке</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>PySide6.QtCharts.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
